File: java/8.5/example2.java
import java.util.Scanner;

public class example2 {
    public static void main(String[] args) {
        Integer storedValue = 100;
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a number");
        Integer valueInput = Integer.parseInt(scanner.nextLine());
        if (valueInput > storedValue) {
            System.out.println("It is more than 100");
        }
    }
}

File: rust/src/8.5/example2.rs
fn main() {
    let mut valueinputs = String::new();
    let storedvalue = 100;
    println!("Enter a number");
    std::io::stdin().read_line(&mut valueinputs).unwrap();
    let valueinput = valueinputs.trim().parse::<i32>().unwrap();
    if valueinput > storedvalue {
        println!("It is more than 100");
    }
}